src/oc/web/components/fullscreen_post.cljs

```diff
@@ -333,7 +333,7 @@
                :disabled (zero? (count (:headline activity-editing)))
                :class (when @(::entry-saving s) "loading")}
               "SAVE"]
-            (tile-menu activity-data dom-element-id "bottom"))]]
+            (more-menu activity-data dom-element-id {:tooltip-position "left"}))]]
```

Comment:
I didn't realize this was suppose to change to the more menu for desktop.  Maybe I missed that in some of the designs.

Reply:
I still don't see the designs in abstract that say to use this menu vs the time menu. @bago2k4